Header header
float64 lat
float64 lon
float32 alt
float32 yaw
float32 roll
float32 pitch